### ISSUE TITLE
Counts not updating on frontend

### ISSUE THREAD

**Issue by hereliesvince** [Shelob9/blockswp-share-block#14]
Neither the counts, or email icon are displaying on the Frontend currently.

**Backend**
![screenshot 2018-01-03 20 28 42_preview](https://user-images.githubusercontent.com/3044814/34546888-08f8e542-f0c5-11e7-94fa-523070a2d9da.png)

**Frontend**
![screenshot 2018-01-03 20 30 50_preview](https://user-images.githubusercontent.com/3044814/34546911-275a163c-f0c5-11e7-873f-874799353284.png)

**Comment by Shelob9** [comment]
The library we're using doesn't expose Twitter counts. Pretty sure that's an API limitation. No way to fix that.

Email not showing in front-end, probably either the showEmail Gutenberg attribute not saving right or the initialization script for front end doesn't parse it from saved atts to props for the shareview component properly.

**Comment by Shelob9** [comment]
Most likely issue is in this conditional here https://github.com/Shelob9/blockswp-share-block/blob/master/front-end.js#L17

**Comment by easilyamused** [comment]
We know some networks no longer share their share counts.

Spent about 4 hours tonight working on this and found a handful of problems. Could not get the plugin to work locally or on pantheon. Worked on fixing the errors from the other ticket, but it seemed to make things worse. 

Did discover that hiding the icons does not render on the font-end nor are the counts changing anymore.

On my local only 2 of the networks showed up because I had a older version (the release from master) When I installed the newer one(after complying) all the new styles worked but still only 2 networks. Worried the old settings are getting stuck in WP. 

To be clear, we had a lot of issues tonight and we would start over with fresh code from master when trying different local setups and dev environments to make sure it was not on our ends. 

We did run the build scrips, Vincent tried it too on Pantheon with same issues of counts not changing or settings rendering on the front-end.
